public/templates/public/components/base.html

```diff
@@ -1,18 +1,22 @@
 {% load staticfiles %}
+{% load render_bundle from webpack_loader %}
 
 <!DOCTYPE html>
 <html>
 	<head>
 		<meta charset="utf-8">
 
 		<title>{% block title %}{% endblock %}</title>
+		{% render_bundle 'main' 'css' %}
 	</head>
 
 	<body>
+		<div id="root"></div>
```

Comment:
How about using an `id` that's more descriptive, such as `react-mount`? That'll help us remember what the `<div>`'s purpose is.

Reply:
👍